bpf/gotracer/go_sql.c

```diff
@@ -110,6 +309,42 @@ int obi_uprobe_queryReturn(struct pt_regs *ctx) {
 
         __builtin_memcpy(&trace->conn, &invocation->conn, sizeof(connection_info_t));
 
+        u64 driver_conn_ptr = invocation->driver_conn_ptr;
+        int hostname_extracted = 0;
```

Comment:
I think we can simply this by always initialising the hostname to '\0'. Maybe extract this code into a function  and return from error scenarions, but also put print statement so we know if we need to debug this.